dll/shellext/shellbtrfs/CMakeLists.txt

```diff
@@ -6,6 +6,9 @@ include_directories(${REACTOS_SOURCE_DIR}/drivers/filesystems/btrfs)
 
 spec2def(shellbtrfs.dll shellbtrfs.spec)
 
+# Don't use C++17 on the ONLY module in the OS that uses namepsace std
```

Comment:
```suggestion
# Don't use C++17 on the ONLY module in the OS that uses namespace std
```
By the way, why does this causes problems?

Reply:
https://github.com/reactos/reactos/blob/master/dll/shellext/shellbtrfs/shellext.h#L62

shellbtrfs decides to use the entire std namespace conflicting with all of our headers byte definitions

Reply:
On C++17 there is a std::byte and since that piece of code uses "using namespace std" which puts the entire standard lib into global namespace (`std::byte` can then be used/accessed with `byte`) this causes name collision with the byte typedef and the compiler can't figure out which one to use